include/inexor/vulkan-renderer/wrappers/instance.hpp

```diff
@@ -0,0 +1,67 @@
+#pragma once
+
+#include "inexor/vulkan-renderer/availability_checks.hpp"
+
+#include <spdlog/spdlog.h>
+#include <vulkan/vulkan.h>
+
+#include <cassert>
+#include <memory>
+#include <optional>
+#include <string>
+
+namespace inexor::vulkan_renderer::wrapper {
```

Comment:
`wrapper` does not match the folder name `wrappers`

Reply:
ah right.

Reply:
done.